pycontrails/core/flight.py

```diff
@@ -11,6 +11,7 @@
 import numpy as np
 import numpy.typing as npt
 import pandas as pd
+import pwlf
```

Comment:
I would suggest only requiring this as an optional dependency. So do the import inside of the `fit_altitude` function. When we do this thing, we try to give a nice error message. So something like:

```python
try:
   import pwlf
except ModuleNotFoundError as e:
   raise ModuleNotFoundError("The 'fit_altitude' function requires the 'pwlf' package. This can be installed with ...") from e
```